docs/Clearwater_Architecture.md

```diff
@@ -49,7 +49,7 @@ As described above, Vellum is used to maintain all long-lived state in the dedpl
 - [Cassandra](http://cassandra.apache.org/).  Cassandra is used by Homestead to store authentication credentials and profile information when an HSS is not in use, and is used by Homer to store MMTEL service settings.  Vellum exposes Cassandra's Thrift API.
 - [etcd](https://github.com/coreos/etcd).  etcd is used by Vellum itself to share clustering information between Vellum nodes and by other nodes in the deployment for shared configuration.
 - [Chronos](https://github.com/Metaswitch/chronos).  Chronos is a distributed, redundant, reliable timer service developed by Clearwater.  It is used by Sprout and Ralf nodes to enable timers to be run (e.g. for SIP Registration expiry)  without pinning operations to a specific node (one node can set the timer and another act on it when it pops).  Chronos is accessed via an HTTP API.
-- [Memcached](https://memcached.org/) / [Astaire](https://github.com/Metaswitch/astaire).  Vellum also runs a Memcached cluster fronted by Astaire.  Astaire is a service developed by Clearwater that enabled more rapid scale up and scale down of memcached clusters. This cluster is used by Sprout for storing registration state, Ralf for storing session state and Homestead for storing cached subscriber data.
+- [Memcached](https://memcached.org/) / [Astaire and Rogers](https://github.com/Metaswitch/astaire).  Vellum also runs a Memcached cluster fronted by Rogers, with synchronization provided by Astaire.  Astaire is a service developed by Clearwater that enabled more rapid scale up and scale down of memcached clusters. This cluster is used by Sprout for storing registration state, Ralf for storing session state and Homestead for storing cached subscriber data. Rogers is a proxy which sits in front of a cluster of memcached instances to provide replication of data and topology hiding. Astaire and Rogers work together to ensure that all data is duplicated across multiple nodes, to protect against data loss during a memcached instance failuue
```

Comment:
typo in `failure` at the end, and missing full stop

Also, just a style point, but I'd prefer if these sentences were rearranged:

> [Memcached](https://memcached.org/) / [Astaire and Rogers](https://github.com/Metaswitch/astaire).  Vellum also runs a Memcached cluster fronted by Rogers, with synchronization provided by Astaire. This cluster is used by Sprout for storing registration state, Ralf for storing session state and Homestead for storing cached subscriber data. Astaire is a service developed by Clearwater that enabled more rapid scale up and scale down of memcached clusters. Rogers is a proxy which sits in front of a cluster of memcached instances to provide replication of data and topology hiding. Astaire and Rogers work together to ensure that all data is duplicated across multiple nodes, to protect against data loss during a memcached instance failure.